usersimcrs/run_simulation.py

```diff
@@ -327,9 +327,8 @@ def load_rasa_diet_classifier(
 
     agent_intents_str: Set[str] = set()
     for v in intent_schema["user_intents"].values():
-        intents = v.get("expected_agent_intents", [])
-        if intents:
-            agent_intents_str.update(intents)
+        intents = v.get("expected_agent_intents", []) or []
```

Comment:
I'm wondering if the or part is needed? When would that be triggered?

Reply:
As of now the `or` is needed because some intents have the key `expected_agent_intents` but no value, thus it returns the get returns None that throws an error when updating the set. 
I believe the interaction model's configuration should be revised to remove this behaviour (part of issue #131).